src/Utils.js

```diff
@@ -125,6 +125,31 @@ function arraysEqual(match, left, right) {
   return left.length === right.length && left.every((el, i) => match(el, right[i]));
 }
 
+export function childrenToSimplifiedArray(nodeChildren) {
+  const childrenArray = childrenToArray(nodeChildren);
+  const simplifiedArray = [];
+
+  for (let i = 0; i < childrenArray.length; i++) {
+    const child = childrenArray[i];
+    const simplifiedChild = simplifiedArray.pop();
+
+    if (simplifiedChild === undefined) {
```

Comment:
can you re-write this with `else if` to get rid of `continue` and appease the linter?
eg.
```js
    if (simplifiedChild === undefined) {
      simplifiedArray.push(child);
    } else if (isTextualNode(child) && isTextualNode(simplifiedChild)) {
      simplifiedArray.push(simplifiedChild + child);
    } else {
      simplifiedArray.push(simplifiedChild);
      simplifiedArray.push(child);
    }
```